backend/controllers/Event.js

```diff
@@ -50,6 +50,16 @@ const createEvent = async (req, res) => {
  * res: response object to return status 200 and events array.
  */
 
+const getAllEvents = async (req, res) => {
+  try {
+    const allEvents = await User.find();
```

Comment:
I am sorry that this is not correct Phuoc.
When you use User.find(), it will return an array ( or object , I am not sure =)) ) of all users 'information, not all events of that users.

To do that, firstly you need to take the user id from the req user id. Let's say, for example, if my user id is 123, you will only need to fetch event of user that has id 123. In my decoded token function, I already extract the token to the id and assign all user information to request. So here, the code to take the id is req.user._id . 

Then, using that information, you will have to loop over the EventParticipation schema to find any event participation of that user with status "accepted". At this point, you will have access to the event's id. That is when you use this event id to find the event you need in the Event schema.  Here, I think you shoul skip over events that have status "completed" ( or finished, i forgot the enum =))) )
You then can store the event in an array and  return respone. 

Yeah so this is probably not so easy if you did not have experience working with backend before, so if at any point you find it confusing, ping me and we will have a quick call asap.

@hoatuyet423 ( just tag you bc i want to make sure you also understand the flow of this controller :3 )